kartotherian/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM debian:buster
+FROM debian:bullseye
```

Comment:
Lets use the wikimedia debian image:
https://docker-registry.wikimedia.org/bullseye/tags/

Reply:
I temporary keep debian:bullseye from docker-hub since image from docker-registry.wikimedia.org doesn't install.